HW_1/Task_1.py

```diff
@@ -0,0 +1,19 @@
+# Задача 1:
+# Найдите сумму цифр трехзначного числа.
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+# Первый вариант решения через числа:
+
+number = int(input("Введите число: "))
+sum = 0
+while number > 0:
+    sum += number % 10
+    number = number//10
```

Comment:
Здесь тоже можно записать, короче.
```suggestion
    number //= 10
```